docs/contribute.rst

```diff
@@ -172,6 +172,12 @@ Helpful commands
     >>> to_checksum_address("0x9c78ee466d6cb57a4d01fd887d2b5dfb2d46288f")
     '0x9C78EE466D6Cb57A4d01Fd887D2b5dFb2D46288f'
 
+Working with the develop branch
+=================================
+
+The big changes to the code all happen in the ``develop`` branch. Those might include changes to the schema both in the user database and the global database. Errors related to partial migrated databases ight manifest as errors in the UI when executing queries or failures to start the app or sign in. For the develop experience instead of the normal ``data`` folder we use another in the same path called ``develop_data`` and contains both users' db and the ``global_data`` folder.
```

Comment:
```suggestion
The big changes to the code all happen in the ``develop`` branch. Those might include changes to the schema both in the user database and the global database. Errors related to partially migrated databases might manifest as errors in the UI when executing queries or failures to start the app or sign in. For working on develop instead of the normal ``data`` rotki directory we use another in the root path called ``develop_data``.
```